pom.xml

```diff
@@ -39,15 +39,28 @@
     <properties>
         <project.scala.version>2.11.12</project.scala.version>
         <project.scala.binary.version>2.11</project.scala.binary.version>
-
-        <neo4j.version>3.5.12</neo4j.version>
-        <driver.version>1.7.5</driver.version>
-        <spark.version>2.4.0</spark.version>
+        <neo4j.version>4.0.1</neo4j.version>
+        <driver.version>4.0.1</driver.version>

```

Comment:
4.0.2 ?

Reply:
ok

Reply:
The last one on maven central is:
https://mvnrepository.com/artifact/org.neo4j.driver/neo4j-java-driver

Reply:
oh sorry